src/parser/core/Combatant.ts

```diff
@@ -509,7 +509,7 @@ class Combatant extends Entity {
         ability: {
           abilityIcon: buff.icon.replace('.jpg', ''),
           guid: buff.ability,
-          name: spell ? spell.name : undefined,
+          name: spell ? spell.name : "undefined",
           type: 0,
```

Comment:
this should very much be undefined without ""

Reply:
Yes, it requires that i set name as nullable, would that be preffered?